### DISCUSSION TITLE
Is there a way to check if a phone number is a WhatsApp contact?

### DISCUSSION THREAD

**Discussion by lucioufba** [Auties00/WhatsappWeb4j#418] [Q&A] [answered]
Hi,
Is there a way to check if a phone number is a WhatsApp contact?
Thanks!

**Comment by EDMPhoenix** [comment]


**Reply by lucioufba** [reply]
Hi @EdmTechPhoenix ,

Thanks for the answer.

But, I want to know how to do this via the whatsappweb4j API, using java.

**Reply by lucioufba** [reply]
Hi @EdmTechPhoenix , 
Thank you very much. I'm really happy with your help!

**Wich version of WhatsappWeb4j did you use in this example code?**
I didn't find these classes in the version 3.5.1 (suggested in the current readme file) :

```

        <dependency>
            <groupId>com.github.auties00</groupId>
            <artifactId>whatsappweb4j</artifactId>
            <version>3.5.1</version>
        </dependency>

```

![image](https://github.com/Auties00/Cobalt/assets/17599123/fffc6bbe-e718-4718-9934-3fdb2b4506a9)

**Reply by Auties00** [reply]
These replies are clearly generated using an llm, please ignore them

**Comment by Auties00** [comment, marked as answer]
there's a method in the WhatsApp class called hasWhatsapp

**Reply by lucioufba** [reply]
I found it here now,
thank you very much!

**Comment by thmHassan** [comment]
whats the daily limit for this via one API and can we do multiple deployments? also how to connect with you? I am currently developing solution for my firm where i might have to deploy multiple instances of whatsapp running at the same time and yours seemed reliable
